tests/server/functional_tests/datasets/test_get_record.py

```diff
@@ -112,7 +112,7 @@ def test_get_record_by_id(mocked_client, task, expected_record_class):
 
     assert response.status_code == 200
     record = expected_record_class.parse_obj(response.json())
-    assert record.id == record_id
+    assert record.id == str(record_id)
```

Comment:
We do convert here. Is it not cleaner to do this in the record definition?

Reply:
Here in the test, the provided record id is defined as an integer. I kept it like this just to validate that the integer valuer is still supported, even if the found record has the string representation of the id.